HedgeLib/Sets/SetObject.cs

```diff
@@ -15,6 +15,8 @@ public class SetObject
 
         public string ObjectType;
         public uint ObjectID;
+        public bool IsTemplateExists = true;
+        public byte[] RawParamData;
```

Comment:
I'm sorry, but I don't like the idea of every SetObject having this array even if it's completely unnecessary (for example, with Gens sets, where template data is not even needed).

Could this be changed to an object within the SetObject's "CustomData" dictionary instead?
For example:
```csharp
obj.CustomData["RawParamData"]
```

instead of:
```csharp
obj.RawParamData
```
?

That way it would only have to be present if the object is using it